trading_calendars/country_calendar_france.py

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import time
+
+from pandas.tseries.holiday import Holiday
+from pytz import timezone
+
+from .common_holidays import (
+    new_years_eve,
+    new_years_day,
+    GoodFriday,
+    EasterMonday,
+    whit_monday,
+    european_labour_day,
+    christmas_eve,
+    christmas,
+    boxing_day,
+)
+from .trading_calendar import HolidayCalendar, TradingCalendar, WEEKDAYS
+
+NewYearsEveBefore2002 = new_years_eve(end_date='2002', days_of_week=WEEKDAYS)
```

Comment:
do you need to explicitly pass WEEKDAYS for all these holidays? If they fall on a weekend, nothing will happen, right?

Reply:
Actually it looks to only affect the half days (New Years Eve/Christmas Eve). I'll remove the rest.

Reply:
gotcha - does it make weekends seem like half days if you don't pass WEEKDAYS?